doc/lua_api.txt

```diff
@@ -2233,6 +2233,7 @@ and `minetest.auth_reload` call the authetification handler.
 * `minetest.find_node_near(pos, radius, nodenames)`: returns pos or `nil`
```

Comment:
'search centre' needs to be included here in square brackets to show it is optional, like:
```
* `minetest.set_mapgen_setting(name, value, [override_meta])`
```